### ISSUE TITLE
unit tests warning

### ISSUE THREAD

**Issue by rs028** [AtChem/AtChem2#360]
When running `make unittests`, this warning message is output. The tests pass anyways.

```
src/solverFunctions.f90:181.51:

    real(kind=DP) :: q(getNumberOfGenericComplex())
                                                   1
Warning: Unused variable 'q' declared at (1)
```

**Comment by spco** [comment]
Yep, this is for the function parser. It's not yet used, but won't do any harm. When compiling, the compiler will just remove it, so there's no computational cost.

**Comment by spco** [comment]
This should now be fixed by #370, as this code no longer exists in the same form. Let me know if you see it still.
